src/uct/ib/mlx5/gga/gga_mlx5.c

```diff
@@ -930,6 +930,13 @@ uct_ib_mlx5_gga_md_open(uct_component_t *component, const char *md_name,
         goto out_free_dev_list;
     }
 
+    if (!(md->super.super.cap_flags & UCT_MD_FLAG_EXPORTED_MKEY)) {
```

Comment:
can we do it in uct_ib_mlx5_gga_check_device?
if not we probably need to revert uct_ib_mlx5_gga_check_device altogether, if we already move some checks to md_open